doc/tutorials/content/gpu_octree.rst

```diff
@@ -68,11 +68,11 @@ K Nearest Neighbors search
 
 .. note::
 
-    The K nearest neighbor algorithm is designed to identify a given (k) number of nearest neighbors to a query point. However, the GPU KNN search algorithm is currently restricted to k=1 only.
+    The K nearest neighbor algorithm is designed to identify a given (k) number of nearest neighbors to a query point. However, the GPU KNN search algorithm is currently restricted to k=1 only. This restriction will be lifted in a future release.
 
 .. note::
 
-    The GPU K nearest neighbor algorithm differs from it's CPU counterpart in implementation. Specifically, the CPU algorithm will only find approximate nearest neighbors, where the user can set the upper bound of the error allowed. In contrast, the GPU algorithm finds the absolute nearest neighbor. Unless it is necessary to find the absolute nearest neighbor, users can utilize approximate nearest search instead, as both functions offer similar functionality until support is added for K>1.
+    The GPU K nearest neighbor algorithm differs from it's CPU counterpart in implementation. Specifically, the CPU algorithm will only find approximate nearest neighbors, where the user can set the upper bound of the error allowed. In contrast, the GPU algorithm finds the absolute nearest neighbor. For K = 1, if an approximation will suffice, users can utilize approximate nearest search instead.
```

Comment:
>  For K = 1, if an approximation will suffice, users can utilize approximate nearest search instead.

The word *should* is paramount here.

Reply:
shall I also mention the reason, i.e `For K = 1, if an approximation will suffice, users should utilize the more efficient approximate nearest search method instead. `

Reply:
definitely.